src/lib.rs

```diff
@@ -826,12 +826,7 @@ impl<'ctx> Bindings<'ctx> {
 /// Determines whether the given cursor is in any of the files matched by the
 /// options.
 fn filter_builtins(ctx: &BindgenContext, cursor: &clang::Cursor) -> bool {
-    let (file, _, _, _) = cursor.location().location();
-
-    match file.name() {
-        None => ctx.options().builtins,
-        Some(..) => true,
-    }
+    !cursor.is_builtin() || ctx.options().builtins
```

Comment:
nit: I'd reverse the order here since the second is way less expensive to check.